setup.py

```diff
@@ -73,7 +73,7 @@ def _read_file(filename):
         'AccessControl >= 4.2',
         'Acquisition',
         'BTrees',
-        'Chameleon >= 3.7.0',
+        'Chameleon >= 3.7.2',
```

Comment:
This is a minor concern, but I would only update the pin in setup.py if it prevents real errors because pins in setup.py are a big blunt hammer. This PR "only"  solves a convenience issue.

Reply:
I have reset `setup.py` to its former state. Depending on how `Zope` is installed, the added test may fail (if `chameleon < 3.7.2` is used).